### DISCUSSION TITLE
GPU Error

### DISCUSSION THREAD

**Discussion by zsz0811** [AMReX-Codes/amrex#3715] [Q&A] [answered]
Hi,
 when running AMReX on GPU, we get this error "/amrex/Src/Base/AMReX_GpuLaunchFunctsG.H line 885: named symbol not found"
any suggestions on what might be wrong?
thank you

**Comment by WeiqunZhang** [comment, marked as answer]
You might need to set `CUDA_ARCH` explicitly. See for example, https://github.com/AMReX-Astro/Castro/pull/2715/files

**Reply by zsz0811** [reply]
Thank you so much. I worked.
